### ISSUE TITLE
Ascend: pow/rsqrt fail bishengir-compile with 'unsupported datatype for arith::ExtFOp to hfusion'

### ISSUE THREAD

**Issue by lvyufeng** [FlagOpen/FlagGems#6226]
## Summary

On Ascend, the FlagGems kernels for `pow` and `rsqrt` fail to compile.
`bishengir-compile` aborts with an LLVM fatal error while lowering to hfusion:

```
[ConvertLinalgRToBinary] encounters error:
LLVM ERROR: unsupported datatype for arith::ExtFOp to hfusion
```

This surfaces as `triton.compiler.errors.MLIRCompilationError` from
`triton/compiler/compiler.py:303`.

`arith::ExtFOp` is a float extension, so the kernels appear to widen a
lower-precision float to a type the hfusion backend does not accept. Both ops
take plain float32 inputs at the call site.

## Environment

- Platform: Ascend 910 (`--target=Ascend910_9382`)
- CANN 9.0.0
- triton-ascend 3.2.2
- Python 3.12
- FlagGems via the FlagOS Torch-FL FlagGems Python path

## Reproduction

```python
import torch_fl, torch

a = torch.randn(4, 4, device="flagos:0").abs() + 0.1
torch.pow(a, 2.0)          # MLIRCompilationError

b = torch.randn(4, 4, device="flagos:0").abs() + 1e-6
torch.rsqrt(b)             # MLIRCompilationError
```

## Full compiler invocation

```
bishengir-compile /tmp/.../kernel.ttadapter.mlir \
  --target=Ascend910_9382 \
  --enable-auto-multi-buffer=True \
  --enable-auto-bind-sub-block=True \
  --enable-auto-blockify-loop \
  --enable-hfusion-compile=true \
  --enable-hivm-compile=true \
  --enable-triton-kernel-compile=true \
  --mlir-print-ir-after-failure \
  --mlir-print-stacktrace-on-diagnostic \
  -o /tmp/.../kernel
```

## Scope

These were the only 2 failures out of 38 tests in the affected CI run, so this is
narrow rather than a broad Ascend breakage. The other FlagGems ops in that run
compile and produce correct results, including `abs`, `all`, `embedding_dense_backward`,
`nll_loss`, `silu_backward`, `slice_backward`, `pow`'s neighbours in the same
dispatch suite, and the factory ops.

## Expected behavior

`pow` and `rsqrt` should compile on Ascend, or the unsupported extension should be
avoided in the kernel for this target.

## Workaround

Routing `pow.Scalar`, `pow.Tensor_Scalar`, `pow.Tensor_Tensor`, `rsqrt` and
`rsqrt_` to the native aclnn kernels on Ascend. Both ops have native
implementations there, so this is a fallback rather than a loss of coverage.

Happy to test a fix on real 910 hardware.
